modules/oc-mirror-updating-cluster-manifests-v2.adoc

```diff
@@ -29,6 +29,20 @@ In oc-mirror plugin v2, the IDMS and ITMS files cover the entire image set, unli
 $ oc apply -f <path_to_oc_mirror_workspace>/working-dir/cluster-resources
 ----
 
+. If you mirrored release images, apply the release image signatures to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f working-dir/cluster-resources/signature-configmap.json 
+----
++
+[IMPORTANT]
+====
+If you are mirroring Operators instead of clusters, do not run the above command. Running the command results in an error because there are no release image signatures to apply.
```

Comment:
```suggestion
If you are mirroring Operators instead of clusters, do not run the preceding command. Running the command results in an error because there are no release image signatures to apply.
```
According to IBM SG, use either "preceding" or "previous".